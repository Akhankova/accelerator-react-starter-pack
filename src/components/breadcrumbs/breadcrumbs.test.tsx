import { render } from '@testing-library/react';
import Breadcrumbs from './breadcrumbs';

describe('Component: Breadcrumbs', () => {
  it('should render correctly', () => {
    const {container} = render(<Breadcrumbs />);
    expect(container).toMatchSnapshot();
  });
});
