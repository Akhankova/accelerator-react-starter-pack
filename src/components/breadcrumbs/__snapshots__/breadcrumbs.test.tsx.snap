// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Breadcrumbs should render correctly 1`] = `
<div>
  <ul
    class="breadcrumbs page-content__breadcrumbs"
  >
    <li
      class="breadcrumbs__item"
    >
      <a
        class="link"
        href="./main.html"
      >
        Главная
      </a>
    </li>
    <li
      class="breadcrumbs__item"
    >
      <a
        class="link"
        href="/"
      >
        Каталог
      </a>
    </li>
  </ul>
</div>
`;
